engine/namegen.ml

```diff
@@ -404,14 +404,13 @@ let rename_bound_vars_as_displayed avoid env c =
 
 let h_based_elimination_names = ref false
 
-let use_h_based_elimination_names () =
-  !h_based_elimination_names && Flags.version_strictly_greater Flags.V8_4
```

Comment:
Note that I am not suggesting to change the option value in any way. If the option has always been disabled and if it ever gets removed, then it is the disabled value that would be used, obviously. So the naming model would certainly not change due to deprecation.